### PR TITLE
Piccy Reactions

### PR DESCRIPTION
**Pull request by JakasaurusRex**
## Description
This pull request adds the last planned stretch goal for now for Piccy, reactions. Reactions are gifs that you can add to peoples post as a quick response to it. Features include:
- [X] Clicking on a reaction button on a Piccy to bring up the gif selection menu. 
- [X] Selecting a reaction brings you back to the home screen where your reaction will have been added to the Piccy.
- [X] Clicking on the icon again will bring you into the reactions page where you can see everyone else who reacted to the Piccy.

## Milestones
This is the last stretch feature I want to implement before I start refactoring my code and fixing major bugs and UI issues. 

## Test Plan
Theres not really a lot of edge cases with this one besides being able to post multiple reactions which I fixed early. But I will be on the lookout for more while testing. 

Screen Shot 2022-07-29 at 2 26 19 PM

### CODE REVIEW

**Comment by JakasaurusRex** [comment]
Included in this PR is a lot of other random changes that I never PR'd because they were never complete including the share to instagram and twitter pod which i installed and uninstalled.

File: Piccy/Piccy/View Controllers/Friends Page/FriendsViewController.m [hunk 1]
```diff
@@ -372,12 +372,12 @@ - (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInte
 }
 
 //Query for the friend requests tab
--(void) requestQuery:(NSString *)container {
+-(void) requestQuery:(NSString *)container withLimit:(int) limit {
```
Comment (sipingggg) [inline Piccy/Piccy/View Controllers/Friends Page/FriendsViewController.m:375]: why you need to have the withLimit: ? seems like all the value you passed in is 10.
